### DISCUSSION TITLE
What does rule.name mean? (i’m a new student)

### DISCUSSION THREAD

**Discussion by eliechah** [Security-Onion-Solutions/securityonion#11155] [Unsupported Versions]
I’m a university student just getting started on Security Onion and they are expecting us to learn this software on our own. I have made progress but I CANNOT find anywhere on what each category or term means. I have learnt a few of them but what does rule.name mean?

Is it like alerts from detection engines for malicious contents or something?

**Comment by dougburks** [comment]
`rule.name` is the name of the rule that generated the alert.

For example, here is an Emerging Threats NIDS rule from https://docs.securityonion.net/en/2.3/managing-alerts.html:
```
alert tcp $HOME_NET any -> $EXTERNAL_NET !1433 (msg:"ET POLICY Outbound MSSQL Connection to Non-Standard Port - Likely Malware"; flow:to_server,established; content:"|12 01 00|"; depth:3; content:"|00 00 00 00 00 00 15 00 06 01 00 1b 00 01 02 00 1c 00|"; distance:1; within:18; content:"|03 00|"; distance:1; within:2; content:"|00 04 ff 08 00 01 55 00 00 00|"; distance:1; within:10; flowbits:set,ET.MSSQL; classtype:bad-unknown; sid:2013409; rev:3;)
```
Looking at the `msg` field, we see `ET POLICY Outbound MSSQL Connection to Non-Standard Port - Likely Malware`. So if Suricata detects traffic that matches what's shown in the rule, then it will fire an alert for `ET POLICY Outbound MSSQL Connection to Non-Standard Port - Likely Malware` and that will show up in our interfaces as `rule.name`.

For more information, please see:
https://docs.securityonion.net/en/2.3/alert-data-fields.html
https://docs.securityonion.net/en/2.3/managing-alerts.html
https://docs.securityonion.net/en/2.3/rules.html
https://docs.securityonion.net/en/2.3/suricata.html
